fdbserver/fdbserver.actor.cpp

```diff
@@ -1107,6 +1108,23 @@ struct CLIOptions {
 private:
 	CLIOptions() = default;
 
+	void parseEnvInternal() {
+		for (std::string knob : getEnvironmentKnobOptions()) {
+			std::vector<std::string> kv;
+			boost::split(kv, knob, [](char c) { return c == '='; });
+			if (kv.size() != 2 || !kv[0].length() || !kv[1].length()) {
+				fprintf(stderr,
+				        "Error: malformed environment knob option: %s%s\n",
+				        ENVIRONMENT_KNOB_OPTION_PREFIX,
+				        knob.c_str());
+				flushAndExit(FDB_EXIT_ERROR);
```

Comment:
The typical behavior for setting an invalid knob via command line is to log a warning and continue without the knob. This avoids concerns where an incorrectly specified knob can bring down a cluster. We may want to consider that here, too.

Reply:
Ok